### ISSUE TITLE
See demo?

### ISSUE THREAD

**Issue by DanielBarstow** [windows-on-earth/winearth-webui#26]
How can I see a demo of the current status?

thanks

Dan Barstow

**Comment by AlexNguyenJJ** [comment]
Hi Dan,

Currently, the new features are not available in a demo until changes have been merged in other repositories (winearth-api and winearth-deploy). To see the current changes in dev, you will need to have all 3 repos on your local computer and set them to the correct branches as they have not been merged into main yet; the simplest thing to do is to wait until those changes have been merged into main and then deployed on the production site.

Thank you for your patience,
Alex Nguyen

**Comment by DanielBarstow** [comment]
Sure. I can wait.

**Comment by kevincoakley** [comment]
Hi Dan, 

The current work in progress is always available at http://winearth.sdsc.edu/ we are meeting on Tuesdays and Thursdays to review the code before merging the code to the live server, so those are the best days to look for changes.  If you don't see any changes to the website UI it is likely the changes were done to the api or the deployment which are not visible to viewers of the site. You are always welcome to provide feedback to what is currently available at http://winearth.sdsc.edu/ .

**Comment by DanielBarstow** [comment]
Thanks Kevin and Alex

**Comment by kevincoakley** [comment]
Hi Dan,

Do you have any updates from Amy about the Optical Flow code for the movies?

**Comment by DanielBarstow** [comment]
Yes, I got an update yesterday.  Unfortunately, she is close, but not done.  Even when done, it would need installation and testing on your end.  And I don't think you have enough time in your work scope to do that.

So. . .I recommend we just stick with the 50 or so movies you already have in place.  That is all we need for the demo and proof of concept.  You have them in two versions - fast and real-speed, and the images look fine.  We'll hold off on Amy's method until UCSD gets the funding for the full project implementation.

OK?

**Comment by kevincoakley** [comment]
@DanielBarstow, 

The website was updated with searching and sorting on the main page. We added the metadata to the movies page. We will be working on adding the map images next week. Let me know if you have any feedback.

**Comment by AlexNguyenJJ** [comment]
@DanielBarstow,

The website was updated with the new changes that were requested. Will be working on the next things for the Final Milestone, which includes fixing the movie display to not flicker when toggling between movie speeds and making working links for sharing. Let me know if you have any feedback.
